examples/typescript/cp/cp-example.ts

```diff
@@ -0,0 +1,7 @@
+import * as k8s from '@kubernetes/client-node';
+
+const kc = new k8s.KubeConfig();
+kc.loadFromDefault();
+
+const cp = new k8s.Cp(kc);
+cp.cpFromPod('default', 'nginx-4217019353-9gl4s', 'nginx', './test.txt', '/tmp');
```

Comment:
Maybe a dumb question here, but how to catch the error for `cp.cpFromPod or cpToPod`, I tried with then().catch, and even combine with try catch block, but failed to capture the error, but get `UnhandledPromiseRejectionWarning:`
```
    try {
        cp.cpToPod(
            "notexistsnamespace",
            podName,
            CONTAINER,
            filename,
            targetDir
        )
        .then(() => {
            console.log("success");
        }).catch((err) => {
            console.error(err);
        })
    } catch (err) {
        console.error(err);
    }
```

Reply:
https://til.hashrocket.com/posts/aucwkd8b36-asyncawait-unhandledpromiserejectionwarning#:~:text=Follow%20on%20Twitter-,Async%2FAwait%20UnhandledPromiseRejectionWarning,await%20you%20get%20an%20UnhandledPromiseRejectionWarning%20.&text=There%20are%20two%20ways%20to,with%20a%20try%20catch%20block.&text=The%20second%20is%20by%20handling%20the%20error%20on%20the%20promise%20itself.

Think it will be helpful to you